en/computer_vision/collision_prevention.md

```diff
@@ -30,16 +27,38 @@ The user is notified through *QGroundControl* while *Collision Prevention* is ac
 
 ## PX4 (Software) Setup
 
-Set the following [parameter](../advanced_config/parameters.md) in *QGroundControl*:
+Enabled collision prevention by [setting the following parameter](../advanced_config/parameters.md) in *QGroundControl*:
 
-* [MPC_COL_PREV_D](../advanced_config/parameter_reference.md#MPC_COL_PREV_D) - Set the minimum allowed distance (the closest distance that the vehicle can approach the obstacle). Set negative to disable *collision prevention*.
+* [MPC_COL_PREV_D](../advanced_config/parameter_reference.md#MPC_COL_PREV_D) - Set the minimum allowed distance (the closest distance that the vehicle can approach the obstacle).
+  Set negative to disable *collision prevention*.
 
   This should be tuned for both the *desired* minimal distance and likely speed of the vehicle.
 
+If you are using a distance sensor attached to your flight controller for collision prevention, it will need to be [attached and configured](#rangefinder) as described in the next section.
+If you are using a companion computer to provide obstacle information see [companion setup](#companion).
+
+
+## PX4 Distance Sensor {#rangefinder}
+
+At time of writing PX4 allows you to use the [Lanbao PSK-CM8JL65-CC5](../sensor/cm8jl65_ir_distance_sensor.md) IR distance sensor for collision prevention "out of the box", with minimal additional configuration:
+- First [attach and configure the sensor](../sensor/cm8jl65_ir_distance_sensor.md), and enable collision prevention (as described above, using `MPC_COL_PREV_D`).
+- Set the sensor orientation using [SENS_CM8JL65_R_0](../advanced_config/parameter_reference.md#SENS_CM8JL65_R_0).
+
+<!-- ROTATION_FORWARD_FACING - Does it matter what angles? - ie is collision prevention active in 3 D? -->
```

Comment:
@mrivi 
1. What is the field of collision prevention? is it just a single field in front of just one sensor in horizontal 2 d space, or does it cover 3d space (in front in a horizontal and vertical plane?)
2. Given that, are there orientations that make more or less sense? Reason I'm asking is that forward facing sensor makes best sense for range traveling horizontally - I'm wondering if a backwards facing one is useful, or if one at 45 degrees just uses the forward component (implying you need twice the range for the same distance of avoidance).

Reply:
It's 360 around the vehicle. So if you have a distance sensor backwards, it will refrain the vehicle from moving backwards if there is an obstacle in that direction. 
The prevention is 2D, so you will never fly over/under stuff